wandb/sdk/launch/runner/kubernetes_runner.py

```diff
@@ -532,7 +538,9 @@ def _inject_defaults(
             launch_project.override_entrypoint is not None,
         )
 
-        env_vars = get_env_vars_dict(launch_project, self._api)
+        env_vars = get_env_vars_dict(
+            launch_project, self._api, MAX_ENV_LENGTHS[self._type]
+        )
```

Comment:
I think it might be a tiny bit cleaner to pass in the `_type` and have the `MAX_ENV_LENGTHS` be internal to the get_env_vars_dict function.

Reply:
you would only have to import `MAX_ENV_LENGTHS` once, and now you don't have to trace an int through three functions.

Reply:
As a global, users can override these values when they don't agree with their system. (Specifically for bare metal launch agents)